content/applications/inventory_and_mrp/inventory/management/products/strategies.rst

```diff
@@ -2,75 +2,113 @@
 How to select the right replenishment strategy
 ==============================================
 
-Minimum Stock rules and Make to Order have similar consequences but
-different rules. They should be used depending on your manufacturing and
-delivery strategies.
+Reordering Rules and Make to Order have similar consequences: creating a
+Purchase or Manufacturing order automatically.
+But those methods work differently. They should be used depending on your 
+manufacturing and delivery strategies.
 
 Terminology
 ===========
 
-Minimum stock rule 
--------------------
+Replenishment Report and Reordering Rules
+-----------------------------------------
 
-Minimum Stock rules are used to ensure that you always have the minimum
+The replenishment report is a list all your products having a negative forecast 
```

Comment:
`The replenishment report is a list all your products having a negative forecast` change to `The replenishment report is a list of all your products that have a negative forecast`

Reply:
done